src/h3lib/lib/h3Index.c

```diff
@@ -846,15 +848,15 @@ void H3_EXPORT(h3ToGeo)(H3Index h3, GeoCoord* g) {
  * @param h3 The H3 index.
  * @param gb The boundary of the H3 cell in spherical coordinates.
  */
-void H3_EXPORT(h3ToGeoBoundary)(H3Index h3, GeoBoundary* gb) {
+void H3_EXPORT(cellToBoundary)(H3Index h3, CellBoundary* gb) {
```

Comment:
Not required, but we might want to change `gb` here and possibly elsewhere (likewise in the doc comment)

Reply:
I can do that.